sources/src/App.vue

```diff
@@ -52,12 +52,12 @@ export default {
               {
                 header: "CLEAR",
                 text:
-                  '<p> Robot Framework has a modular architecture that can be extended with bundled and self-made <a href="#libraries">test libraries</a>. </p> <p> Test data is defined in files using the syntax shown in the <a href="#examples">examples</a> below. A file containing test cases creates a test suite and placing these files into directories creates a nested structure of test suites. </p>'
+                  '<p> Robot Framework has a modular architecture that can be extended with bundled and self-made <a href="#libraries">libraries</a>. </p> <p> Data is defined in files using the syntax shown in the <a href="#examples">examples</a> below. A file containing tasks or test cases creates a suite, and placing these files into directories creates a nested structure of suites. </p>'
```

Comment:
- Changing "test library" to "library" here and elsewhere is good.
- Changing "test suite" to "suite is fine as well.
- I don't like changing "test data" to "data" that much. Somehow "Data is defined in files ..." sounds a bit strange to my ear. I agree we cannot really use "test data" and "test or task data" would be bad as well. If nobody has better ideas, perhaps we god with just "data".
- Changing "test cases" to "tasks or test cases" is not ok as it changes bias too much to task automation. I'd be fine with "tests or tasks", though, and have used that in the User Guide. Please change this here and elsewhere.